include/sysexits.h

```diff
@@ -35,9 +35,11 @@
 /*
  *  SYSEXITS.H -- Exit status codes for system programs.
  *
- *	This include file attempts to categorize possible error
- *	exit statuses for system programs, notably delivermail
- *	and the Berkeley network.
+ *	This include file attempts to categorize 4.4BSD error exit
+ *	statuses for system programs, notably delivermail (which
+ *	became sendmail) and the Berkeley network. These values
+ *	have been deprecated in the base system since FreeBSD 8.0,
+ *	see style(9).
```

Comment:
I'm unsure about the deprecation notice should be here. But if it is, something more like:

This exists only for interface compatibility, but are deprecated for FreeBSD base software.